### ISSUE TITLE
invicti professional edition v24.3.0 / .nss file have issues / An exception error occurred during the report import:["'utf-8' codec can't decode byte 0xc6 in position 106: invalid continuation byte"]

### ISSUE THREAD

**Issue by johnfelipe** [DefectDojo/django-DefectDojo#10076]
Pls review this video:

*An exception error occurred during the report import:["'utf-8' codec can't decode byte 0xc6 in position 106: invalid continuation byte"]*

https://github.com/DefectDojo/django-DefectDojo/assets/428820/c6f4e26d-d2ca-4599-8ecc-c1c1406f329d

**Comment by manuel-sommer** [comment]
Could you please upload a sample file to reproduce the issue? Then, we can close it with a PR

**Comment by manuel-sommer** [comment]
@johnfelipe , please share a sample file here.

**Comment by johnfelipe** [comment]
https://gist.github.com/johnfelipe/59ba7758276ddaab4971207367712e85

is censured like HCL Appscan

**Comment by manuel-sommer** [comment]
Netsparker supports only JSON format, but you provided xml content...

**Comment by johnfelipe** [comment]
invicti professional edition v24.3.0

this is former netsparker, but im use invicti, can we put new scanner
called Invicti professional or something?

**Comment by manuel-sommer** [comment]
Sure, why not. Feel free to make a PR. 👍

**Comment by johnfelipe** [comment]
Then for make you PR, do u need JSON for netspeaker?

**Comment by valentijnscholten** [comment]
@johnfelipe Can you summarize your request as I am not fully understanding it. Is that you want the existing invicti parser  at https://github.com/DefectDojo/django-DefectDojo/tree/master/unittests/scans/invicti to support the XML format from https://gist.github.com/johnfelipe/59ba7758276ddaab4971207367712e85? Or is it a different scanner, like a "Professional Edition"?
